### DISCUSSION TITLE
Question about installing the MUMPS linear solver

### DISCUSSION THREAD

**Discussion by 0xF0F0F0F** [coin-or/Ipopt#542] [Q&A] [answered]
When I install the MUMPS in MSYS2, should I just using 
"pacman -S mingw-w64-x86_64-mumps" 
OR
following the instruction and getting MUMPS from "https://github.com/coin-or-tools/ThirdParty-Mumps.git"?

**Comment by svigerske** [comment, marked as answer]
The instructions were written with the idea in mind that they should be followed.

But there may be a way to also use the MUMPS package from the MSYS2 package repository. You'll need to set appropriate `--with-mumps-*` flags.
